h2o-algos/src/main/java/hex/glm/GLMModel.java

```diff
@@ -552,6 +562,12 @@ public GLMWeightsFun(Family fam, Link link, double var_power, double link_power,
       _link = link;
       _var_power = var_power;
       _link_power = link_power;
+      _oneMinusVarPower = 1-_var_power;
+      _twoMinusVarPower = 2-_var_power;
+      _oneOoneMinusVarPower = _var_power==1?1:1.0/(1-_var_power);
+      _oneOtwoMinusVarPower = _var_power==2?1:1.0/(2-_var_power);
+      _oneOLinkPower = 1.0/_link_power;
+      _oneOLinkPowerSquare = _oneOLinkPower*_oneOLinkPower;
       _theta = theta;
```

Comment:
This is purely for speed up as we need to do this for every row.  I just calculate them once and use them for every row afterwards.  Probably just some small speedup unless the dataset is huge.

Reply:
No need to re-calculate it per each row, when it can be done just once :)